contracts/airdrops/NFTDrop.sol

```diff
@@ -117,16 +110,4 @@ contract NFTDrop is ERC721, AirdropInfo, AirdropMerkleProof, Ownable {
     function getAirdropAmount() external view returns (uint256) {
```

Comment:
///

Reply:
This is actually a deprecated contract, we are keeping it for a "just in case" scenario so we're not really working on it